src/assignmenttwo/Contact.cs

```diff
@@ -0,0 +1,77 @@
+﻿// <copyright file="Contact.cs" company="PlaceholderCompany">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+
+namespace Contact
+{
+    using System.Collections.Generic;
+    using System.Xml.Linq;
+
+    /// <summary>
+    /// Represents a contactsList with name, phone number, email and note.
+    /// </summary>
+    public class Contact
+    {
+        // Represents a global identifier for genarating unique values.
+        private static int id = 1;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Contact"/> class.
+        /// </summary>
+        /// <param name="name">The name of the contactsList.</param>
+        /// <param name="number">The phone number of the contactsList.</param>
+        /// <param name="email">The email of the contactsList.</param>
+        /// <param name="notes">The note of the contactsList.</param>
+        public Contact(string name, string number, string email, string notes)
+        {
+            this.Name = name;
+            this.Number = number;
+            this.Email = email;
+            this.Note = notes;
+            this.GlobalId = id;
+
+            // Increments id for next Contact instance.
+            id++;
+        }
+
+        /// <summary>
+        /// Gets or sets the name of the contactsList.
```

Comment:
This is not the name of the contactsList instead this is the name of the contact.
Please update these accordingly wherever required.
```suggestion
        /// Gets or sets the name of the contact.
```